res/css/views/settings/encryption/_EncryptionCard.pcss

```diff
@@ -30,6 +30,13 @@
             text-align: center;
         }
     }
+
+    /* extra class for specifying that we don't need a border */
+    &.mx_EncryptionCard_noBorder {
```

Comment:
my CSS fu is weak. Why specifically `&.mx_EncryptionCard_noBorder` rather than just `.mx_EncryptionCard_noBorder`?

Reply:
The `&` means that the class applies to the same element as the enclosing selector.  So, with the `&`, it means that it matches elements that are both `.mx_EncryptionCard` and `.mx_EncryptionCard_noBorder`, whereas without the `&`, it would match elements that are `.mx_EncryptionCard_noBorder` that are descendants of `.mx_EncryptionCard`.